backend/pyproject.toml

```diff
@@ -22,7 +22,7 @@ emails = "^0.6"
 fastapi-users = "9.3.2"
 httpx-oauth = "^0.10.1"
 PyYAML = "^6.0"
-
+apprise = {git = "https://github.com/caronc/apprise.git", rev = "master"}
```

Comment:
Is it really necessary to setup the dependency from the Git repository?

Reply:
Regarding the apprise dependency. Master branch was used because recent changes are required that have not been included in a release yet. Specifically: https://github.com/caronc/apprise/pull/655#event-7472744743 and [WIP] https://github.com/caronc/apprise/issues/657